spec.html

```diff
@@ -74,10 +74,7 @@ <h1>Runtime Semantics: PerformPromiseAny ( _iteratorRecord_, _constructor_, _res
           1. Set _iteratorRecord_.[[Done]] to *true*.
           1. Set _remainingElementsCount_.[[Value]] to _remainingElementsCount_.[[Value]] - 1.
           1. If _remainingElementsCount_.[[Value]] is 0, then
-            1. Let _errorsArray_ be CreateArrayFromList(_errors_).
-            1. Let _error_ be a newly created `AggregateError` object.
-            1. Perform ! CreateDataProperty ( _error_, `"errors"`, _errorsArray_ ).
-            1. Return ThrowCompletion ( _error_ ).
+            1. Return ThrowAggregateError(_errors_).
```

Comment:
Should I add `!` here?

```suggestion
            1. Return ! ThrowAggregateError(_errors_).
```

Reply:
it returns an abrupt completion, so you definitely can not use `!`.

Reply:
Ok, thank you for explanation